files/haproxy.init

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+#
+# chkconfig: - 85 15
+# description: HA-Proxy is a TCP/HTTP reverse proxy which is particularly suited \
+#              for high availability environments.
+# processname: haproxy
+# config: /etc/haproxy/haproxy.cfg
+# pidfile: /var/run/haproxy.pid
+
+# Script Author: Simon Matter <simon.matter@invoca.ch>
+# Version: 2004060600
```

Comment:
another possibility is to use something like pleaserun to generate the file, but that would be a hard dependency on an external module that is still very much in flux.